ext/dom/element.c

```diff
@@ -1137,7 +1248,7 @@ PHP_METHOD(DOMElement, hasAttribute)
 
 	DOM_GET_OBJ(nodep, id, xmlNodePtr, intern);
```

Comment:
To be consistent, use `ZEND_THIS` directly instead of having the `id` zval?

Reply:
Do you want me to make these changes to untouched code? I prefer to do it in a follow up then.

Reply:
That's fair I don't mind it to be a follow-up (or even a clean-up before)